src/config/wmodules-config.c

```diff
@@ -134,12 +134,16 @@ int Read_WModule(const OS_XML *xml, xml_node *node, void *d1, void *d2)
         mwarn("A deprecated Vulnerability Detector configuration block was found. It will be ignored.");
         OS_ClearNode(children);
         return 0;
-    } else if (!strcmp(node->values[0], WM_AZURE_CONTEXT.name)) {
+    }
+#endif
```

Comment:
Do not place the `endif` here and create a new `ifndef` in line 145. Put both `WM_VULNDETECTOR_CONTEXT` and `WM_KEY_REQUEST_CONTEXT` "if" conditions in the same `ifndef`-`endif` block and then add the `WM_AZURE_CONTEXT` condition after that block, outside the `ifndef CLIENT`

Reply:
Done in e4ca2d8.